### DISCUSSION TITLE
Print A3 sized PDF to A4 paper not working as expected

### DISCUSSION THREAD

**Discussion by BFjimmy** [sumatrapdfreader/sumatrapdf#4606] [General]
I try to print our blue prints at work but they end up only print an A4 portion of the A3 PDF onto the paper. Same printer settings in PDF24 works as intended. Anyone with suggestions on what I do wrong?

**Comment by GitHubRulesOK** [comment]
Nothing "wrong" just PDF24 is using personal users Artifex GhostScript as its print engine and
SumatraPDF is totally dependent on Windows "native" print as image which is default to US letter or ISO A4 depending what system default is?

The page size needs to be fit print by shrink A3 to A4.

So it depends what printer settings Windows uses when SumatraPDF is started it will usually be the default A4 unless you enforce a change in page size when printing. Here is mine on Windows 10 where it should be ISO A4 but Windows insists this device is Atlantic half American British !!
![image](https://github.com/user-attachments/assets/f2e2d50f-6fdb-4841-b39a-ba56c531881b)

For me I need to alter to A4 and fit
![image](https://github.com/user-attachments/assets/5a3169e5-3f63-4d70-b344-de53a5c064c5)
![image](https://github.com/user-attachments/assets/b4bcf0b6-38ad-478c-88b2-e2e854f356e7)

**Comment by BFjimmy** [comment]
Ah, OK. I will try to find my settings that will work 'cause we want to continue using SumatraPDF.
